xlators/features/locks/src/clear.c

```diff
@@ -175,6 +177,15 @@ clrlk_clear_posixlk(xlator_t *this, pl_inode_t *pl_inode, clrlk_args *args,
                               plock->user_flock.l_len != ulock.l_len))
                 continue;
 
+            if (!setlk_interrupt)
+                goto no_interrupt;
+
+            if (strcmp(plock->client_uid, client_uid) != 0 ||
+                plock->client_pid != client_pid) {
+                continue;
+            }
+
+        no_interrupt:
```

Comment:
nit: we can easily avoid the label and check the pid before the client to make it more efficient in many cases
```suggestion
            if (setlk_interrupt) {
                if ((plock->client_pid != client_pid) ||
                    (strcmp(plock->client_uid, client_uid) != 0)) {
                    continue;
                }
            }
```